Formula/l/loyalsoldier-geoip.rb

```diff
@@ -0,0 +1,39 @@
+class LoyalsoldierGeoip < Formula
+  desc "Convenient tool to merge, convert and lookup IP & CIDR"
+  homepage "https://github.com/Loyalsoldier/geoip"
+  url "https://github.com/Loyalsoldier/geoip/archive/refs/tags/202407190813.tar.gz"
+  sha256 "9ba285d9801e33a03fea4b76a52eb766ff78e0e4d7c92b42a86999c25dd5a689"
+  license any_of: ["CC-BY-SA-4.0", "GPL-3.0-or-later"]
+  head "https://github.com/Loyalsoldier/geoip.git", branch: "master"
+
+  depends_on "go" => :build
+  conflicts_with "geoip", because: "both install `geoip` binaries"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "-o", bin/"geoip"
+  end
+
+  def caveats
+    <<~EOS
+      Usage examples:
+      #{"  "}
+        merge CIDRs from local file and print to stdout:
+        $ cat redundant-cidr.txt | geoip merge
+      #{"  "}
+        merge CIDRs from local file and only print IPv4 CIDRs to stdout:
+        $ cat redundant-cidr.txt | geoip merge -t ipv4
+      #{"  "}
+        merge CIDRs from remote URL and print to stdout:
+        $ curl -s https://core.telegram.org/resources/cidr.txt | geoip merge
+      #{"  "}
+        merge CIDRs from remote URL and only print IPv6 CIDRs to stdout:
+        $ curl -s https://core.telegram.org/resources/cidr.txt | geoip merge -t ipv6
+    EOS
+  end
```

Comment:
This is not Homebrew specific and should be in upstream documentation

Reply:
Removed.